rudder-core/src/main/scala/com/normation/rudder/services/policies/DataStructures.scala

```diff
@@ -378,6 +378,13 @@ final object Policy {
     path.replaceFirst(subPath, s"${p.technique.id.name.value}/${p.id.getRudderUniqueId}")
   }
 
+  /*
+   * Replace the Rudder Unique Id also in the path for input list
+   */
+  def replaceRudderUniqueId(path: String, p: Policy): String = {
+    path.replace(TAG_OF_RUDDER_MULTI_POLICY, p.id.getRudderUniqueId)
```

Comment:
it should be a `replaceAll` to avoid problems in a case where someone put several tag in the path for whatever reason